awx/main/models/ha.py

```diff
@@ -293,8 +293,8 @@ def refresh_capacity_fields(self):
             self.cpu_capacity = 0
             self.mem_capacity = 0  # formula has a non-zero offset, so we make sure it is 0 for hop nodes
         else:
-            self.cpu_capacity = get_cpu_effective_capacity(self.cpu)
-            self.mem_capacity = get_mem_effective_capacity(self.memory)
+            self.cpu_capacity = get_cpu_effective_capacity(self.cpu, is_execution_node=bool(self.node_type in (Instance.Types.EXECUTION,)))
```

Comment:
as Alan had suggested, the keyword arg should probably be called `is_control_node`, thus we should check that self.node_type is in `(Instance.Types.CONTROL,Instance.Types.Hybrid)`